RealmSwift/Combine.swift

```diff
@@ -57,6 +57,18 @@ extension ObjectKeyIdentifiable {
     }
 }
 
+@available(macOS 10.15, iOS 13, tvOS 13, watchOS 6, *)
+extension Projection: Identifiable, Hashable {
+    /// A stable identifier for this projection. This value will be
+    /// the same for all projections of this object
```

Comment:
This doesn't seem true? Two different projections of the same object will have different ObjectIdentifiers. Should the  `id` be derived from the object key?